code/modules/vehicles/cm_armored.dm

```diff
@@ -123,27 +123,29 @@ var/list/TANK_HARDPOINT_OFFSETS = list(
 
 //No one but the gunner can gun
 //And other checks to make sure you aren't breaking the law
-/obj/vehicle/multitile/root/cm_armored/tank/handle_click(var/mob/living/user, var/atom/A, var/list/mods)
+/obj/vehicle/multitile/root/cm_armored/tank/handle_click(mob/living/user, atom/A, list/mods)
 
-	if(!can_use_hp(user)) return
+	if(istype(A,/obj/screen) || A == src || mods["middle"] || mods["shift"] || mods["alt"])
```

Comment:
this should be in code/_onclick/

Reply:
I'll handle this part after this PR or #550 are merged.